tests/integration/records/table_validator.py

```diff
@@ -12,6 +12,7 @@
     expected_table2table_column_types
 )
 from records_mover.records import DelimitedVariant
+from .mover_test_case import MoverTestCase
```

Comment:
This file, one of the 3 largest in the project, increased in size to 398 lines.  The total size of those files is now 3440 lines (target: 897).  Is this file complex enough to refactor?